R/RunDiagnostics.R

```diff
@@ -152,7 +152,11 @@ getDefaultCovariateSettings <- function() {
 #'                                   This expression can use the variables 'cohortId' and 'seed'.
 #'                                   Default is "cohortId * 1000 + seed", which ensures unique identifiers
 #'                                   as long as there are fewer than 1000 cohorts.
-
+#'                                   
+#' @param useAchilles                Logical. Should the pre-computed Achilles analyses be used to get concept counts? TRUE or FALSE (default)
```

Comment:
At least a link to achilles somewhere in the documentation would be great here as not all users will have experience with it.

Reply:
added.